src/parser.js

```diff
@@ -48,8 +48,8 @@ function getLastExpression(code) {
   const minified = (code || '')
     // remove comments
     .replace(/\/\*[\s\S]*?\*\/|\/\/.*/g, '')
-    // remove all white space
-    .replace(/\s/g, '');
+    // remove all white space outside quotes
+    .replace(/[ ]+(?=[^"]*(?:"[^"]*"[^"]*)*$)/g, '');
```

Comment:
This only works when using `"` as quote character though. Javascript has 3 valid quote characters that can be used:

```js
screen.getByPlaceholderText(`how should I call you?`)
screen.getByPlaceholderText('how should I call you?')
screen.getByPlaceholderText("how should I call you?")
```

Can you expand the regex to deal with those cases as well?